sdk/keyvault/keyvault-secrets/src/secretsModels.ts

```diff
@@ -214,43 +206,54 @@ export interface UpdateSecretOptions {
    */
   notBefore?: Date;
   /**
-   * @member {Date} [expires] Expiry date in UTC.
+   * @member {Date} [expiresOn] Expiry date in UTC.
    */
-  expires?: Date;
+  expiresOn?: Date;
   /**
    * @member {{ [propertyName: string]: string }} [tags] Application specific
    * metadata in the form of key-value pairs.
    */
   tags?: { [propertyName: string]: string };
-  /**
-   * @member {coreHttp.RequestOptionsBase} [requestOptions] Options for this request
-   */
-  requestOptions?: coreHttp.RequestOptionsBase;
```

Comment:
Sorry if this is me being forgetful, but why are we removing this and re-flattening the options? I remember us changing it to this way as a result of users studies showing this might be helpful.

Reply:
I couldn't recall rationale, and it seems good to be super consistent with how to apply the options in RequestOptionsBase, especially the abortSignal. I found it strange to put `abortSignal` under `requestOptions` when this isn't the case anywhere else in the SDK.

That said, if someone can dig up direct feedback on this point, we can be consistent in the other direction?

Reply:
Do you remember what user study showed this? This is the general direction that all of the TypeScript SDKs are moving towards though. If you remember the details, we should probably rise them to the rest of the team asap.